source/documentation/tools.md

```diff
@@ -41,6 +41,102 @@ For Python:
 
 **Note on support:** the Analytical Platform team does not offer support on the topic of packaging. DASD users are encouraged to use their #r, #conda and #python Slack channels to support each other, or ask your line manager about training. Of course, if there is something broken with the platform itself, or something unique about the platform that prevents you from installing a library, then of course do raise it with the team in the normal ways.
 
+## R's install.packages()
+
+**NB Only use this method for playing - use Conda for project work.**
+
+You can install R packages from the R Console:
+
+```R
+install.packages("ggplot2")
+```
+
+This will find the latest version of the package in CRAN and install it in: `~/R/library`.
+
+However this method is pretty basic. Refer to the tips in the following sections.
+
+### Package version incompatible with R version
+
+Often if you try to install the latest version of a package, it will require a more recent version of R than you have:
+
+```bash
+> install.packages("text2vec")
+Installing package into ‘/home/davidread/R/library’
+(as ‘lib’ is unspecified)
+Warning in install.packages :
+  package ‘text2vec’ is not available (for R version 3.5.1)
+```
+
+There are a few options to avoid this:
+
+Solution 1: AP may have a newer version of RStudio tool which might have the version of R needed. To upgrade, see: [Managing your analytical tools](managing-your-analytical-tools)
+
+Solution 2: Use [conda](#conda) - it's recommended for use with Analytical Platform in general. It works out which version is compatible with your R version (make sure you run this in the Terminal):
+
+```bash
+conda install r-test2vec
+```
+
+Solution 3: Specify a version that is compatible with your R version. e.g. at https://www.rdocumentation.org/packages/text2vec look at the "depends" field for the R version it requires. Change the version (drop-down at the top) to go back to see how it changes for older releases. You can see that text2vec 6.0 requires R (>= 3.6.0), but text2vec 5.1 requires only R (>= 3.2.0).
+
+```R
+devtools::install_version('text2vec', version='0.5.1')
```

Comment:
do we assume they've got/know how to get `devtools`?

Reply:
It's on the list of packages we install in conda environment by default, so I think that's good for the vast majority